libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/route/RouteFetcher.java

```diff
@@ -49,12 +50,29 @@ public RouteFetcher(Context context, String accessToken) {
     routeUtils = new RouteUtils();
   }
 
+  // Package private (no modifier) for testing purposes
+  RouteFetcher(Context context, NavigationRoute navigationRoute) {
+    this.contextWeakReference = new WeakReference<>(context);
+    this.navigationRoute = navigationRoute;
+    this.accessToken = "pk.xx";
```

Comment:
What about passing the `accessToken` as parameter instead of hardcoding it here?